cosmos-proxy/src/tidoopfs.js

```diff
@@ -39,7 +41,88 @@ function fileExists() {
     } // if else
 } // fileExists
 
+function checkConfFile() {
+    var validConfFile = true;
+
+    if (conf.host === undefined) {
```

Comment:
In addition to `undefined`, I would check about the emptyness of the paramters (i.e., it exists, but no value has been configured).